Remove 3.1.0-specific words from the README (#4168)
OAI/OpenAPI-Specification: README.md

This GitHub project is the starting point for OpenAPI. Here you will find the information you need about the OpenAPI Specification, simple examples of what it looks like, and some general information regarding the project.

## Versions

> This repository contains full specification documents for [all published OpenAPI versions](versions). For release notes and RC versions, refer to the [releases page](releases).

Reviewer: ```suggestion
This repository contains [the Markdown sources](versions) for [all published OpenAPI Specification versions](https://spec.openapis.org/). For release notes and RC versions, refer to the [releases page](releases).
```

> This repository contains [the Markdown sources](versions) for [all published OpenAPI Specification versions](https://spec.openapis.org/). For release notes and RC versions, refer to the [releases page](releases).

Reviewer 2: What are "RC versions"?

Reviewer: Release Candidates, see https://spec.openapis.org/oas/v3.1.1.html#appendix-a-revision-history

> This repository contains [the Markdown sources](versions) for [all published OpenAPI Specification versions](https://spec.openapis.org/). For release notes and RC versions, refer to the [releases page](releases).

Reviewer 2: ```suggestion
This repository contains [the Markdown sources](versions) for [all published OpenAPI Specification versions](https://spec.openapis.org/). For release notes and release candidate versions, refer to the [releases page](releases).
```

The OpenAPI Initiative encourages participation from individuals and companies alike. If you want to participate in the evolution of the OpenAPI Specification, consider taking the following actions:

> * Review the [specifications](versions).

Reviewer 3: ```suggestion
* Review the specification [markdown sources](versions) and [authoritative _source-of-truth_ HTML renderings](https://spec.openapis.org/).
```
